src/main/java/util/ConnectionPool.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Properties;
+
+
+public class ConnectionPool {
+
+    private static HikariDataSource ds;
+
+    static{
+        HikariConfig config = new HikariConfig();
+
+        Properties prop=new Properties();
+        try(InputStream input=ConnectionPool.class.getClassLoader().getResourceAsStream("db.properties")){
+            prop.load(input);
+        } catch(IOException e){
+           e.printStackTrace();
+        }
+        config.setJdbcUrl(prop.getProperty("db.url"));
```

Comment:
formatting issues

Reply:
Thanks for catching those formatting issues! In future, could you kindly summarize these formatting concerns into one comment? That way, I can fix all instances in one go. Much appreciated!